stylesheets/facts.css

```diff
@@ -0,0 +1,39 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    width: 1400px;
+    height: 600px;
+    justify-content:center;
+    }
+
+body {
+    background-image: url('../images/planter.jpg');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 100vh;
+    margin: 0;
+    } 
+
+li {
+    padding: 10px;
+} 
+
+h1 { 
+    margin-bottom: 0;
+    padding-top: 50px;
+    padding-bottom: 0;
+    font-family:Papyrus ;
```

Comment:
We should be able to remove this line. Since we set `font-family` to Papyrus in the shared `styles.css` file, the value should be inherited if we import both `styles.css` and `facts.css` into the `facts.html` page.